pages/common/install.md

```diff
@@ -0,0 +1,24 @@
+# install
+
+> Copy files and set attributes.
+> Provide a standardized way of manipulating a file/directory's ownership and permissions while copying file or creating directory.
```

Comment:
It really depends on the point-of-view, do you want the common usage or the overall technical capabilities. It still like `cp` with just some additional options, even if its often usage is in Makefiles.

Now the last two commits has all changes you suggested.